packages/apollo-server/src/index.ts

```diff
@@ -31,6 +35,22 @@ export class ApolloServer<Context> extends ExpressServer {
 
     opts.app = express();
     super(opts);
+
+    opts.app.use('/.health-check', (req, res, next) => {
+      if (opts.onHealthCheck) {
+        opts
+          .onHealthCheck(req)
+          .then(() => {
+            res.status(200).send('Healthy');
+          })
+          .catch(() => {
+            res.status(500).send('Unhealthy');
+          });
+      } else {
+        res.status(200).send('Healthy');
```

Comment:
We may want to consider following the specifications in the [draft RFC proposal for health checks](https://tools.ietf.org/html/draft-inadarei-api-health-check-00), particularly observing the format described (in a basic sense, at least) and using `Content-type: application/vnd.health+json`.

Reply:
There was a newer rfc, so I used `Content-type: application/health+json`